subprojects/common/src/main/java/net/covers1624/wt/api/script/runconfig/RunConfig.java

```diff
@@ -101,11 +103,14 @@ default void progArg(String... args) {
      *
      * @param props The properties.
      */
-    void sysProp(Map<String, String> props);
+    void sysProp(Map<String, Object> props);
 
     /**
      * @return Gets the System Properties for this RunConfiguration.
      */
     Map<String, String> getSysProps();
-}
 
+    static String toString(@Nullable Object obj) {
```

Comment:
This should not be part of public API in RunConfig. It should be moved up to DefaultRunConfig